### PR TITLE
Fix formulas for relative AIM volumes

### PR DESCRIPTION
**Pull request by stevenvdb**

### CODE REVIEW

**Comment by tovrstra** [comment]
Thanks. A test is failing: can you change the author name of the commit to match exactly the one in the AUTHORS file? (This gets checked to impose consistency.) It can be done as follows:

- Fix name and e-mail in `~/.gitconfig`
- `git commit --amend --reset-author`
- `git push (remote) (branch) -f

A "force push" is ok for fixing such things in pull requests.

**Comment by stevenvdb** [comment]
On my local machine everything was fine.
The problem was that GitHub overrides this information with name and email address from the GitHub Personal Settings.

On 16-06-16 13:37, Toon Verstraelen wrote:

Thanks. A test is failing: can you change the author name of the commit to match exactly the one in the AUTHORS file? (This gets checked to impose consistency.) It can be done as follows:

  *   Fix name and e-mail in ~/.gitconfig
  *   git commit --amend --reset-author
  *   `git push (remote) (branch) -f

A "force push" is ok for fixing such things in pull requests.

File: horton/part/hirshfeld.py [hunk 1]
```diff
@@ -107,8 +107,8 @@ def do_dispersion(self):
 
             for i in xrange(self.natom):
                 n = self.numbers[i]
-                volumes[i] = radial_moments[i,2]/populations[i]
-                ref_volume = self.proatomdb.get_record(n, 0).get_moment(3)/n
+                volumes[i] = radial_moments[i,3]
```
Comment (tovrstra) [inline horton/part/hirshfeld.py]: From travis-ci:

    horton/part/hirshfeld.py   103    12   W0612 Unused variable 'populations'
    horton/part/hirshfeld.py   110         C0326 Exactly one space required after comma

**Comment by tovrstra** [comment]
Merging and fixing in another PR.

**Pull request merged**